src/components/Checkout/BasketItem.vue

```diff
@@ -75,8 +74,7 @@ export default {
 		return {
 			activateTimer: true,
 			loanVisible: true,
-			loanRes20Version: 'control',
-			setTimedMessage: false,
+			loanRes20Version: 'control'
```

Comment:
I didn't see any other mentions of this loanRes20 exp, I'm thinking I should get rid of this line?